python/nano/tutorial/notebook/training/pytorch/accelerate_pytorch_training_bf16.ipynb

```diff
@@ -0,0 +1,382 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> ⚠️ **Warning**\n",
+    "> \n",
+    "> Using BFloat16 precision with `torch < 1.12` may result in extremely slow training."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "source": [
+    "## Pre-define Model and Dataloader"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "source": [
+    "In this guide, we take the fine-tuning of a [ResNet-18 model](https://pytorch.org/vision/main/models/generated/torchvision.models.resnet18.html) on [OxfordIIITPet dataset](https://pytorch.org/vision/main/generated/torchvision.datasets.OxfordIIITPet.html) as an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Define model and dataloader\n",
+    "\n",
+    "from torch import nn\n",
+    "from torchvision.models import resnet18\n",
+    "\n",
+    "class MyPytorchModule(nn.Module):\n",
+    "    def __init__(self):\n",
+    "        super().__init__()\n",
+    "        self.model = resnet18(pretrained=True)\n",
+    "        num_ftrs = self.model.fc.in_features\n",
+    "        # Here the size of each output sample is set to 37.\n",
+    "        self.model.fc = nn.Linear(num_ftrs, 37)\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        return self.model(x)\n",
+    "\n",
+    "\n",
+    "import torch\n",
+    "from torchvision import transforms\n",
+    "from torchvision.datasets import OxfordIIITPet\n",
+    "from torch.utils.data.dataloader import DataLoader\n",
+    "\n",
+    "def create_train_dataloader():\n",
+    "    train_transform = transforms.Compose([transforms.Resize(256),\n",
+    "                                          transforms.RandomCrop(224),\n",
+    "                                          transforms.RandomHorizontalFlip(),\n",
+    "                                          transforms.ColorJitter(brightness=.5, hue=.3),\n",
+    "                                          transforms.ToTensor(),\n",
+    "                                          transforms.Normalize([0.485, 0.456, 0.406],\n",
+    "                                                               [0.229, 0.224, 0.225])])\n",
+    "\n",
+    "    # apply data augmentation to the train_dataset\n",
+    "    train_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=train_transform, download=True)\n",
+    "\n",
+    "    # prepare data loader\n",
+    "    train_dataloader = DataLoader(train_dataset, batch_size=32, shuffle=True)\n",
+    "\n",
+    "    return train_dataloader"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## A) Subclass TorchNano"
```

Comment:
The prefix of subtitle "Subclass TorchNano" and "Use @nano decorator" are not consistent. You are using:
```markdown
## A) Subclass TorchNano

## B. Use `@nano` decorator
```
Please make them consistent :)

That is:

```markdown
## A) Subclass TorchNano

## B) Use `@nano` decorator
```

Same for other notebooks.